config/initializers/custom_exception_mappings.rb

```diff
@@ -9,12 +9,13 @@
   "ActiveRecord::RecordNotSaved"       => :bad_request,
   "ActiveRecord::RecordInvalid"        => :bad_request,
   "ActionController::ParameterMissing" => :bad_request,
+  "Catalog::ApprovalError"             => :service_unavailable,
+  "Catalog::ConflictError"             => :conflict
   "Catalog::InvalidParameter"          => :bad_request,
   "Catalog::NotAuthorized"             => :forbidden,
   "Catalog::OrderUncancelable"         => :bad_request,
-  "Catalog::TopologyError"             => :service_unavailable,
-  "Catalog::ApprovalError"             => :service_unavailable,
-  "Catalog::SourcesError"              => :service_unavailable,
   "Catalog::RBACError"                 => :service_unavailable,
-  "Discard::DiscardError"              => :bad_request
+  "Catalog::SourcesError"              => :service_unavailable,
+  "Catalog::TopologyError"             => :service_unavailable,
+  "Discard::DiscardError"              => :bad_request,
```

Comment:
Re-ordered these (and `lib/exceptions.rb`) to be alphabetical just for aesthetic reasons.